bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```diff
@@ -36,7 +36,7 @@ def agent_stages
         :bosh_micro_go,
         :aws_cli,
         :logrotate_config,
-      ]
+      ].reject{ |s| RbConfig::CONFIG['host_cpu'] == 'powerpc64le' and s ==  :bosh_micro_go }
```

Comment:
(Hmm. I lost my prev comment...)

Given that we skip bosh_micro_go stage we can also skip bosh_ruby which also means we dont have to make it work on ppc. Wdyt?